SB2018/2018-03-23.md

```diff
@@ -0,0 +1,15 @@
+---
+to: All Teams
+subject: Media Consent Forms
+---
+
+Hi all,
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 18, they can fill out the forms themselves, however if students are under 18, it is a legal requirement that a parent/guardian signs the form for them.
+
+These forms must be filled and sent to us to get a ticket for the competition, and to ultimately gain entry.
+
+To submit a ticket just send me an email with an attached PDF of the tickets! We may reply with any errors we see.
```

Comment:
I don't think you mean "ticket" either on this line or the following one.

Reply:
a5b9e06f65f4fc34d860e21a42230aedad76ee79

Reply:
fixed in 2363f601e21f5645280e98d5437df0a7d4308a93